source/sdr/tfa_effects/README.md

```diff
@@ -0,0 +1,5 @@
+# script and data used to generate the image `tfa_effects.png`
+
+The image `tfa_effects.png` is meant to illustrate the effect of the TFA value on the stream, SDR, and mask outputs. It was generated using the script and data in this folder. The data are outputs from the SDR model run on the SDR test dataset. All parameters were set to the defaults, except TFA which was set to 100, 400, and 1000.
+
+Run `python tfa_effects_script.py` from within this directory to re-create the file `tfa_effects.png` (requires `matplotlib`, `numpy`, and `pygeoprocessing`).
```

Comment:
I see that the output TIFFs have been committed here, which is fine since the files themselves are so small, and it does make sense that we might take this approach since the test data isn't a dependency of the build process (and probably shouldn't be).

But I wonder, what would you think about either fleshing out the script to also run SDR the 3 times as is documented, or else creating a second script to run SDR to create these files?  Just thinking about future us and making it easier to update this when there are notable changes to the model.  On the other hand, the graphic is for illustrative purposes and maybe this isn't a useful thing for us to do at this point.

Anyways, what do you think about this?

Reply:
That makes sense! I updated the script to run SDR three times and so we don't need the TIFFs checked in. The final PNG is still checked in, so this won't have to run as part of the build process - just if we ever want to update it.

Reply:
Yep, I agree with this!  We don't want to have to rerun sdr on each UG build, so makes sense to commit the final graphic and we can update that committed graphic when we need to.